lib/packages/core/src/api/factories/message/sendEncryptedTextMessage.ts

```diff
@@ -30,7 +31,7 @@ export const sendEncryptedTextMessage = (service: BurstService):
         recipientId: string,
         recipientPublicKey: string,
         senderKeys: Keys,
-        deadline: number = 1440,
+        deadline: number = DefaultDeadline,
```

Comment:
It's still an optional parameter. You may overwrite it, if you want, otherwise `Default` is applied - jfyi